src/unistd.rs

```diff
@@ -27,7 +29,7 @@ impl Fork {
 
     pub fn is_parent(&self) -> bool {
         match *self {
-            Fork::Parent(_) => true,
+            Fork::Parent {..}  => true,
             _ => false
         }
     }
```

Comment:
pub fn is_parent(&self) -> bool {
        !is_child(self)
    }

while you're at it? :)